pkg/controllers/csi/driver/server.go

```diff
@@ -141,14 +141,20 @@ func (srv *Server) Start(ctx context.Context) error {
 }
```

Comment:
There was a comment left unanswered in the other PR which also sparks my interest tbh:

https://github.com/Dynatrace/dynatrace-operator/pull/4353#pullrequestreview-2574897633

Could someone answer that?

Reply:
`Probe` request is send every few seconds and is not rejected by Server rate limiter so we will see confirmation 
the Server process is running in a given period of time.

Reply:
It would be great to have this explanation inside the comment before the debug log. It will simplify future maintenance IMO.